### PR TITLE
Admin services

### PR DESCRIPTION
**Pull request by Filari02**
created admin services where we can request to update our data files. Do you have any comments?

### CODE REVIEW

**Review by Tomas-Per — COMMENTED** [review]
Currently the parsing activation is manual correct? If so, will there be a way to implement and modify an automatic data updater?

**Comment by Filari02** [comment]
now we activate parsing manually, i believe when we will start using MIF servers, then we could make updater automatic

**Review by Tomas-Per — APPROVED** [review]

File: AdminService/AdminService.cs [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+                else if (command == "0")
+                {
+                    break;
+                }
+
+                else
+                {
+                    Console.WriteLine("Wrong input, use !help to learn about commands");
+                }
```
Comment (Ugniusst) [inline AdminService/AdminService.cs:43]: All if and else if could be changed with case. However, I don't know which way is faster

Comment (Tomas-Per) [reply AdminService/AdminService.cs:43]: That only makes a difference in performance if if statements are nested

**Pull request merged**